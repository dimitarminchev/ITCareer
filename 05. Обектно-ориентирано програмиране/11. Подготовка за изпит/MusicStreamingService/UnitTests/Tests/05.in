AddUser John 30
AddPlaylist John RockHits
AddSongToPlaylist John RockHits Thunderstruck 292 ACDC Rock AlbumSong TheRazorsEdge
AddSongToPlaylist John RockHits EnterSandman 331 Metallica Metal AlbumSong Metallica
AddSongToPlaylist John RockHits HighwayToHell 208 ACDC Rock AlbumSong HighwayToHell
GetTotalDurationOfPlaylist John RockHits
GetSongsByGenreFromPlaylist John RockHits Rock
GetSongsAboveDurationFromPlaylist John RockHits 300
End