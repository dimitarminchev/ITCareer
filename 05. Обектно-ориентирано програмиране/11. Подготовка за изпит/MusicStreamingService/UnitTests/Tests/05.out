Created User John!
Created Playlist RockHits for User John!
Added song Thunderstruck to Playlist RockHits!
Added song EnterSandman to Playlist RockHits!
Added song HighwayToHell to Playlist RockHits!
Total duration of RockHits: 831 seconds
Title: HighwayToHell
Duration: 208 seconds
Artist: ACDC
Album: HighwayToHell
Title: Thunderstruck
Duration: 292 seconds
Artist: ACDC
Album: TheRazorsEdge
Title: EnterSandman
Duration: 331 seconds
Artist: Metallica
Album: Metallica